### PR TITLE
feat: Add option "Purchase on invoice" to Offline Purchase Methods.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes https://github.com/fossasia/open-event-frontend/issues/7758

#### Short description of what this resolves:
This pull req. add the option  Purchase on invoice for offline payment methods.

#### Changes proposed in this pull request:
Screenshot from 2021-09-16 09-39-05

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
@mariobehling hey, please review this pull request.

File: app/models/event.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default class Event extends ModelBase.extend(CustomPrimaryKeyMixin, {
   canPayByStripe  : attr('boolean', { defaultValue: false }),
   isStripeLinked  : attr('boolean'),
   canPayByCheque  : attr('boolean', { defaultValue: false }),
+  canPayByInvoice : attr('boolean', { defaultValue: false }),
```
Comment (maze-runnar) [inline app/models/event.js:66]: you have to do server changes for this.

File: app/models/event.js [hunk 2]
```diff
@@ -71,6 +72,7 @@ export default class Event extends ModelBase.extend(CustomPrimaryKeyMixin, {
   paymentCurrency : attr('string', { defaultValue: 'USD' }),
   paypalEmail     : attr('string'),
   chequeDetails   : attr('string'),
+  invoiceDetails  : attr('string'),
```
Comment (maze-runnar) [inline app/models/event.js:75]: for this too.

File: app/models/event.js (hunk 1 above)
Comment (iamdevvalecha) [reply app/models/event.js:66]: ok

**Review by pc-beast — COMMENTED** [review]
Please post screenshots of invoice and mails too after doing suggested changes in backend.

**Review by pc-beast — APPROVED** [review]

**Review by maze-runnar — APPROVED** [review]

**Pull request merged**